elements/pl-number-input/pl-number-input.mustache

```diff
@@ -15,16 +15,25 @@
             <span class="input-group-text">{{label}}</span>
         </span>
         {{/label}}
-        <input name="{{name}}" type="text" class="form-control" size="35" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
+        <input name="{{name}}" type="text" class="form-control {{^display_append_span}}pl-number-input-force-radius-border{{/display_append_span}}" size="{{size}}" {{^editable}}disabled{{/editable}} {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}} aria-describedby="basic-addon1" placeholder="{{shortinfo}}" />
+        {{#display_append_span}}
         <span class="input-group-append">
-            {{#suffix}}<span class="input-group-text">{{suffix}}</span>{{/suffix}}
+            {{#suffix}}<span class="input-group-text {{^shortinfo}}pl-number-input-force-radius-border{{/shortinfo}}">{{suffix}}</span>{{/suffix}}
+            {{#shortinfo}}
```

Comment:
Just to confirm - is there consensus around the decision to hide the question mark along with the placeholder text on `hide-help-text`?

@mfsilva22 ?

Reply:
100% agree there should be some sort of standardization for these options. 

Right now, `hide-help-text` disables _all_ help prompts on `pl-checkbox` (c.f. https://prairielearn.readthedocs.io/en/latest/elements/#pl-checkbox-element). I added this option due to the need to differentiate between Multiple Choice (MC) and Multiple Selection (MS) questions as students didn't detect subtle differences between checkboxes and radio buttons.

Reply:
Following everyone's advice I've set `hide-help-text` to disable the question mark and `placeholder` to disable the placeholder text inside the input box.